online-members/online-members.css

```diff
@@ -112,3 +112,14 @@
   display: flex;
   align-items: center;
 }
+
+footer {
+  display: block;
+  width: 100%;
+  padding: 8px;
+}
+
+footer .info-repo {
+  font-weight: 100;
+  text-align: center;
+}
```

Comment:
Is there a reason we're styling the footer here differently than css of other pages.